src/components/app_catalog/detail/cluster_picker.js

```diff
@@ -16,30 +16,42 @@ const ClusterList = styled.div`
   overflow: auto;
 `;
 
+const themeBorderRadius = ({ theme }) => css`
+  border-radius: ${theme.border_radius};
+`;
+
+const themeBackgroundShade4 = ({ theme }) => css`
+  background-color: ${theme.colors.shade4};
+`;
+
+const themeColorWhite2 = ({ theme }) => css`
+  color: ${theme.colors.white2};
+`;
+
```

Comment:
@axbarsan I have my doubts here. Changing the syntax to this style everywhere bloats up the code a lot. And I am confused by the fact that I find exactly the style you proposed to me initially, like here:

https://github.com/giantswarm/happa/blob/master/src/components/cluster/detail/RegionAndVersions.js#L12-L20

or here

https://github.com/giantswarm/happa/blob/master/src/components/UI/AppContainer.js#L21-L22

Reply:
Well this is the standard way of using styled components. We could turn to another styling solution, but if we want to keep using styled, we kind of have to use the library the way it was meant to be used